pkg/kinflate/app/application.go

```diff
@@ -26,16 +26,15 @@ import (
 	interror "k8s.io/kubectl/pkg/kinflate/internal/error"
 	"k8s.io/kubectl/pkg/kinflate/resource"
 	"k8s.io/kubectl/pkg/kinflate/transformers"
-	"k8s.io/kubectl/pkg/kinflate/types"
 	"k8s.io/kubectl/pkg/loader"
 )
 
 type Application interface {
 	// Resources computes and returns the resources for the app.
-	Resources() (types.ResourceCollection, error)
```

Comment:
Not a deal breaker but was wondering if we should call it just Collection :) it will then read: `resource.Collection` ?

Reply:
`Collection` sounds like an interface, and `ResourceCollection` sounds like a concrete type.